libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway.hs

```diff
@@ -478,10 +473,13 @@ ratifyStateSpec ::
   IsConwayUniv fn =>
   ConwayRatifyExecContext Conway ->
   Specification fn (RatifyState Conway)
-ratifyStateSpec ConwayRatifyExecContext {crecGovActionMap} =
+ratifyStateSpec _ =
   constrained $ \st ->
-    match st $ \_ _ expired _ ->
-      expired `subset_` dom_ (lit crecGovActionMap)
+    match st $ \_ enacted expired _ ->
+      mconcat
+        [ assert $ enacted ==. lit mempty
+        , assert $ expired ==. lit mempty
```

Comment:
These seem to be overconstraining the state. We want to get cases where there are values in `enacted` and `expired`

Reply:
We discussed this on a call when you were on holiday 😜. Let's discuss in a huddle today?